web-client/public/js/updateapp.js

```diff
@@ -0,0 +1,4 @@
+import grnState from "grnstate";
+export var updateApp = function (grnState) {
+
+}
```

Comment:
Note you don't have to import `grnState` here because it is already passed as a parameter.